substrate/frame/verify-signature/Cargo.toml

```
@@ -0,0 +1,70 @@
+[package]
+name = "pallet-verify-signature"
```

```suggestion
name = "frame-verify-signature-extension"
```

Not really a pallet. Also doesn't really need the `Config` trait etc. This can just be a standalone extension like `CheckMetadataHash` extension.

---

maybe https://github.com/paritytech/polkadot-sdk/pull/3685#discussion_r1786520997

---

Yes, we use the pallet to make use of the benchmarking infrastructure. I guess we could rename it as well in a follow up.